modules/accounting/assets/src/admin/components/transfers/NewTransfer.vue

```diff
@@ -123,7 +123,7 @@ export default {
                 this.showAlert('success', __('Transfer Successful!', 'erp'));
                 this.fetchAccounts();
                 this.resetData();
-                this.$router.push('/transfers');
+                this.$router.push('/settings/banks/transfers');
```

Comment:
Make sure the router is right.

Reply:
yes, I have consulted the index.js file's routes